modules/logging-content-filter-drop-records.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+//
+// * logging/performance_reliability/logging-content-filtering.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="logging-content-filter-drop-records_{context}"]
+= Configuring content filters to drop unwanted log records
+
+When the `drop` filter is configured, the log collector evaluates log streams according to the filters before forwarding. The collector drops unwanted log records that match the specified configuration.
+
+.Prerequisites
+
+* You have installed the {clo}.
+* You have administrator permissions.
+* You have created a `ClusterLogForwarder` custom resource (CR).
+
+.Procedure
+
+. Add configuration for a filter to the `filters` spec in the `ClusterLogForwarder` CR.
```

Comment:
I think it's pretty generally referred to as just spec since that's what this type of section in the CRs is called
![image](https://github.com/openshift/openshift-docs/assets/40172997/58f0fade-470f-4a90-999f-ca8a12b25a62)

Reply:
oh nice I have not seen this one! tbh that's a relief lol